splash/browser_tab.py

```diff
@@ -85,6 +87,10 @@ def _init_webpage(self, verbosity, network_manager, splash_proxy_factory, render
         self.web_view.setPage(self.web_page)
         self.web_view.setAttribute(Qt.WA_DeleteOnClose, True)
 
+        self.set_window_size(render_options.get_window_size())
```

Comment:
removed that, it now initialized browser window & viewport size with `defaults.VIEWPORT_SIZE`